R/R/loonGrob.R

```diff
@@ -1019,7 +1019,7 @@ xy_coords_layer <- function(layer, native_unit = TRUE) {
     )
   } else if(type == "histogram"){
     list(
-      x = l_cget(layer, "x"),
+      x = l_cget(widget, "x"),
```

Comment:
Hi @z267xu,

Did you ever run into this issue? I.e. when setting a breakpoint using `browser()` at this place and running:

```r
library(loon)
h <- l_hist(iris)
loonGrob(h)
```

I never hit his line. This is because `loonGrob.l_layer_histogram` does not call `xy_coords_layer` function. Please feel free to remove this `else if (type == "histogram")` part.